--- origsrc/homebank-3.8/configure.ac	2008-03-31 06:41:04.000000000 -0500
+++ src/homebank-3.8/configure.ac	2008-04-07 22:03:13.406250000 -0500
@@ -45,7 +45,7 @@
     if test x$noofx != xtrue
     then
         AC_CHECK_LIB(ofx, ofx_set_status_cb, OFX_0_7="-DOFX_0_7")
-        LDFLAGS="${LDFLAGS} -lofx"
+        LIBS="${LIBS} -lofx"
         CFLAGS="${CFLAGS} $OFX_0_7"
     else
         noofx=true
